docs/docs/default-actions.mdx

```diff
@@ -230,3 +228,28 @@ takes its presence into account so that [conversation behaviour is customizable]
 `action_unlikely_intent` cannot be included in the training stories. It can **only** be added to rules.
 
 :::
+
+## action_extract_slots
+
+This action runs after each user turn, before the next assistant action prediction and execution.
+`action_extract_slots` loops through the [slot mappings](./domain.mdx#slot-mappings) of each domain slot in order to set or update
+slots throughout the conversation with information extracted from the latest user message.
+
+If `action_extract_slots` finds a [custom slot mapping](./domain.mdx#custom-slot-mappings), it will check first if a custom action was defined in the
```

Comment:
Should I replace with `initially`? If there's no custom action defined, an empty list is returned (or in other words no SlotSet events are added to the tracker because there is no custom action to execute). Then the default action just moves on to check if it needs to run a custom validation action which is explained in the next paragraph.

Reply:
Gotcha, reading it again now it makes more sense so I think it's fine this way